src/hooks/useSubmit/utils.ts

```diff
@@ -87,7 +87,7 @@ export const sign = async ({
     params.chainId ||= chainId || `${walletClient.chain.id}`
 
     if (walletClient?.account?.type === "local") {
```

Comment:
I see that we check this prop in `useSetKeyPair` & `useLinkAddress` too, but I wasn't sure if I can remove that code of we should keep it.

Reply:
I think these checks should be safe to remove 👍